ynr/settings/local.py.container.example

```diff
@@ -0,0 +1,30 @@
+# Only set DEBUG to True in development environments.
+DEBUG = True
+
+# These Postgres settings should match docker-compose.yml.
+DATABASES["default"]["NAME"] = "ynr"  # noqa
+DATABASES["default"]["USER"] = "ynr"  # noqa
```

Comment:
Assuming we're only using testing settings to run tests, you probably don't need this

This didn't work for me. I needed to specify
```py
DATABASES = {
    "default": {
        "ENGINE": "django.db.backends.postgresql",
        "NAME": 'ynr',
        "USER": "ynr",
        "PASSWORD": "ynr",
        "HOST": "dbpsql",
        "PORT": "5432",
    },
}
```
here. This is with `DJANGO_SETTINGS_MODULE=ynr.settings.base` in `env/frontend.env`

Reply:
Resolved by https://github.com/DemocracyClub/yournextrepresentative/commit/fb11f14cd55a5b43f86fb08ce5bd05d17afa05a7 (not yet pushed to this branch).